### ISSUE TITLE
Portable version offline?

### ISSUE THREAD

**Issue by vatterspun** [qTox/qTox#4916]
So there used to be two folders for the portable version:

```
https://qtox-win.pkg.tox.chat/qtox/win32/download-portable
https://qtox-win.pkg.tox.chat/qtox/win64/download-portable
```

Did those move?

Portable version background: https://github.com/qTox/qTox/issues/3929

**Comment by smaragdus** [comment]
@anthonybilinski

It is extremely arrogant to close an issues without responding. I am no longer wondering why people are seeking alternatives to **qTox**.

Since version **1.14.1** Windows archives are obviously unavailable (once these used to be found there- [x32 version](https://build.tox.chat/view/qtox/job/qTox-cmake-stable_build_windows_x86_release/) & [x64 version](https://build.tox.chat/view/qtox/job/qTox-cmake-stable_build_windows_x86-64_release/)). These archives are not available on [releases page](https://github.com/qTox/qTox/releases). Where are they?

**Comment by Diadlo** [comment]
@smaragdus Do you mean closing #5327? It was closing with comment, that there already the same issue in the repo. What's wrong? @anthonybilinski may not know why there is no portable version, but there is no reason to have two same issues

**Comment by vatterspun** [comment]
> ... there is no reason to have two same issues

Agreed, but I think the suggestion is that keeping users in the loop about why the issues were closed (e.g. won't fix / resolved / can't reproduce) discourages future submissions.  

Anyway the main focus was to make sure the instructions up on the qTox www.PortableFreeware.com entry are accurate and correct and it looks like that's the case.

**Comment by anthonybilinski** [comment]
>@anthonybilinski may not know why there is no portable version, but there is no reason to have two same issues

That's what happened. I saw two issues for the same issue so closed the newer of the two to not duplicate tracking, and commented explaining why.

>It is extremely arrogant to close an issues without responding.

I commented agreeing that it's an issue before closing the ticket as a duplicate https://github.com/qTox/qTox/issues/5327#issuecomment-419149192. I don't understand what you're talking about.

>but I think the suggestion is that keeping users in the loop about why the issues were closed (e.g. won't fix / resolved / can't reproduce) discourages future submissions.

Are you saying that I SHOULDN'T keep users in the loop about why issues are closed? IMO keeping users in the loop of why issues are closed is a good thing, which is why I commented on the issue.

I don't really understand what the problem with the ticket management is, but as for the issue of the missing portable build, that seems completely covered by this ticket.

**Comment by smaragdus** [comment]
To make it clear- I once reviewed [qTox](https://qtox.github.io/) at The Portable Freeware [Forum](https://www.portablefreeware.com/forums/viewtopic.php?f=4&t=22422), then someone else ([vatterspun](https://github.com/vatterspun)) added it to The Portable Freeware [Database](https://www.portablefreeware.com/?id=2755). At this time there were Windows archives ([x32](https://build.tox.chat/view/qtox/job/qTox-cmake-stable_build_windows_x86_release/) and [x64](https://build.tox.chat/view/qtox/job/qTox-cmake-stable_build_windows_x86-64_release/)), upon which the instructions how to use **qTox** in portable mode were based. These Windows archives were updated until version 1.14.1. Later these archives did not get any updates and did not appear in [Releases](https://github.com/qTox/qTox/releases) page on GitHub. I suppose that was why **vatterspun** opened this issue. As far as I know this was the first issue opened concerning the problem mentioned above and it was never responded. I am aware that the **qTox** installer can be easily extracted but it is an awkward way to avoid the installer for the average user.

**Comment by vatterspun** [comment]
@anthonybilinski 
> Are you saying that I SHOULDN'T keep users in the loop about why issues are closed?

I mean just the opposite.  With rare exception, is ideal to communicate early and often about moves on open source projects to keep everyone on the same page and thereby encourage participation.

> IMO keeping users in the loop of why issues are closed is a good thing, which is why I commented on the issue.

I actually didn't check the issue link for an associated comment, so that might be on me.  Either way, I consider this resolved and should continue testing qTox and will post here if I have ideas/issues.

Thanks.

**Comment by sudden6** [comment]
We have portable builds in the official releases and nightly builds again.

**Comment by smaragdus** [comment]
@sudden6 

> We have portable builds in the official releases and nightly builds again.

Please add download links for portable builds to [main page](https://github.com/qTox/qTox) and [releases page](https://github.com/qTox/qTox/releases)- it is hard to find the portable builds, the ones I have found:

- [x86](https://build.tox.chat/view/qtox/job/qTox-cmake-stable_build_windows_x86_release/)
- [x86-64](https://build.tox.chat/view/qtox/job/qTox-cmake-stable_build_windows_x86-64_release/)

are very outdated (version **1.14.1**).

**Comment by sudden6** [comment]
Seems I have to correct myself here, there are only portable builds for nightlies: https://github.com/qTox/qTox-nightly-releases/releases
